# 
trajectory_msgs/JointTrajectoryPoint movj
---------------
int32 error_code
